libs/ledger-live-common/src/data/icons/svg/BCERT.svg

```diff
@@ -0,0 +1,13 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<svg id="Layer_1" data-name="Layer 1" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 360 360">
```

Comment:
please remove `id` `data-name` `xmlns` attrs
And add `width="360" height="360"` to the attrs